mayastor/src/bdev/storage/bdev.rs

```diff
@@ -0,0 +1,56 @@
+use std::{collections::HashMap, convert::TryFrom};
```

Comment:
I'm not sure what the intent of this is i think its the bdev:/// thing right? There is an issue for that already I beleive, where the suggestion is to rename that to local:/// or loopback:///

Reply:
Yes this is exactly that. The name of the scheme can be easily changed from bdev to local if that is preferable.

Reply:
This is in an actual fact CAS-1.  Will effect this change by replacing bdev:// with loopback://